code/datums/station_traits/postive_traits.dm

```diff
@@ -0,0 +1,220 @@
+	name = "Premium internals boxes"
+	trait_type = STATION_TRAIT_POSITIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "The internals boxes for your crew have been upsized and filled with bonus equipment."
+	trait_to_give = STATION_TRAIT_PREMIUM_INTERNALS
+
+/datum/station_trait/strong_supply_lines
+	name = "Strong supply lines"
+	trait_type = STATION_TRAIT_POSITIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Prices are low in this system, BUY BUY BUY!"
+	blacklist = list(/datum/station_trait/distant_supply_lines)
+
+
+/datum/station_trait/strong_supply_lines/on_round_start()
+	SSeconomy.pack_price_modifier *= 0.8
+
+/datum/station_trait/filled_maint
+	name = "Filled up maintenance"
+	trait_type = STATION_TRAIT_POSITIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Our workers accidentally forgot more of their personal belongings in the maintenance areas."
+	blacklist = list(/datum/station_trait/empty_maint)
+	trait_to_give = STATION_TRAIT_FILLED_MAINT
+
+	// This station trait is checked when loot drops initialize, so it's too late
+	can_revert = FALSE
+
+/datum/station_trait/quick_shuttle
+	name = "Quick Shuttle"
+	trait_type = STATION_TRAIT_POSITIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Due to proximity to our supply station, the cargo shuttle will have a quicker flight time to your cargo department."
+	blacklist = list(/datum/station_trait/slow_shuttle)
+
+/datum/station_trait/quick_shuttle/on_round_start()
+	. = ..()
+	SSshuttle.supply.callTime *= 0.5
+
+/datum/station_trait/deathrattle_department
+	name = "deathrattled department"
+	trait_type = STATION_TRAIT_POSITIVE
+	show_in_report = TRUE
+	trait_flags = STATION_TRAIT_ABSTRACT
+	blacklist = list(/datum/station_trait/deathrattle_all)
+
+	var/department_to_apply_to
+	var/department_name = "department"
+	var/datum/deathrattle_group/deathrattle_group
+
+/datum/station_trait/deathrattle_department/New()
+	. = ..()
+	deathrattle_group = new("[department_name] group")
+	blacklist += subtypesof(/datum/station_trait/deathrattle_department) - type //All but ourselves
+	report_message = "All members of [department_name] have received an implant to notify each other if one of them dies. This should help improve job-safety!"
+	RegisterSignal(SSdcs, COMSIG_GLOB_JOB_AFTER_SPAWN, PROC_REF(on_job_after_spawn))
+
+
+/datum/station_trait/deathrattle_department/proc/on_job_after_spawn(datum/source, datum/job/job, mob/living/spawned)
+	SIGNAL_HANDLER
+	if(department_to_apply_to)
+		if(!(job.job_department_flags & department_to_apply_to))
+			return
+
+	var/obj/item/implant/deathrattle/implant_to_give = new()
+	deathrattle_group.register(implant_to_give)
+	implant_to_give.implant(spawned, spawned, TRUE, TRUE)
+
+
+/datum/station_trait/deathrattle_department/service
+	name = "Deathrattled Service"
+	trait_flags = STATION_TRAIT_MAP_UNRESTRICTED
+	weight = 1
+	department_to_apply_to = DEP_FLAG_SERVICE
+	department_name = "Service"
```

Comment:
could make all of these use `__DEFINES/department_defines.dm`

Reply:
done